ios/FluentUI/Core/Fonts.swift

```diff
@@ -33,8 +33,8 @@ public final class Fonts: NSObject {
     @objc public static var button2: UIFont { return preferredFont(forTextStyle: .footnote, weight: .medium) }
     /// Medium 10pt - Does not scale automatically with Dynamic Type
     @objc public static let button3 = UIFont.systemFont(ofSize: 10, weight: .medium)
-    /// Medium 15pt - Does not scale automatically with Dynamic Type
-    @objc public static let button4 = UIFont.systemFont(ofSize: 15, weight: .medium)
+    /// Medium 16pt - Does not scale automatically with Dynamic Type
+    @objc public static let button4 = UIFont.systemFont(ofSize: 16, weight: .regular)
```

Comment:
might be out of scope but we are trying to get rid of these static fonts. i think caption1 in XXLarge might the the font you want.

Reply:
I think caption1 resizes automatically. Also it's kind of weird naming to use it for a button. I'd rather wait for the repo to remove all these vars.